raiden/network/transport/matrix/client.py

```diff
@@ -475,7 +481,23 @@ def _sync(self, timeout_ms: int = 30000) -> None:
                 len(room["account_data"]["events"]) for room in response["rooms"]["join"].values()
             ),
         )
-        self._handle_response(response, is_first_sync)
+        before_processing = time.time()
+        self._handle_response(response=response, first_sync=is_first_sync)
+        processing_time_s = time.time() - before_processing
+
+        # If processing the matrix response takes longer than the poll timout, it cannot be
+        # ensured that all messages have been processed.
+        # Therefore an exception is thrown when in development mode.
+        timeout_s = timeout_ms // 1_000
```

Comment:
why don't we just compare the millisecond value?

Reply:
because `time()` returns seconds and there is only ´time_ns()`, which makes less sense in my opinion.